# a set of values that determine which kind(s) of autonomy are being used
# if we determine that only one kind of autonomy can be used at a time, then we should change this to an enum

bool naive